src/components/form/Lookup/Location/LocationFields.tsx

```diff
@@ -1,32 +1,10 @@
-import {
-  DisplayCountryFragment as CountryLookupItem,
-  DisplayRegionFragment as RegionLookupItem,
-  DisplayZoneFragment as ZoneLookupItem,
-} from '../../../../api/fragments/location.generated';
+import { DisplayLocationFragment as LocationLookupItem } from '../../../../api/fragments/location.generated';
 import { LookupField } from '../../index';
-import {
-  useCountryLookupLazyQuery,
-  useRegionLookupLazyQuery,
-  useZoneLookupLazyQuery,
-} from './LocationLookup.generated';
+import { useLocationLookupLazyQuery } from './LocationLookup.generated';
 
-export const CountryField = LookupField.createFor<CountryLookupItem>({
-  resource: 'Country',
-  useLookup: useCountryLookupLazyQuery,
-  label: 'Country',
-  placeholder: 'Search for a country by name',
-});
-
-export const RegionField = LookupField.createFor<RegionLookupItem>({
-  resource: 'Region',
-  useLookup: useRegionLookupLazyQuery,
-  label: 'Region',
-  placeholder: 'Search for a region by name',
-});
-
-export const ZoneField = LookupField.createFor<ZoneLookupItem>({
-  resource: 'Zone',
-  useLookup: useZoneLookupLazyQuery,
-  label: 'Zone',
-  placeholder: 'Search for a zone by name',
+export const LocationField = LookupField.createFor<LocationLookupItem>({
+  resource: 'Location',
+  useLookup: useLocationLookupLazyQuery,
+  label: 'Location',
+  placeholder: 'Search for a location by name',
```

Comment:
We'll still need the Region & Zone Fields. Probably just prefixed with "Field" now. Feel free to further split them out into other files since they are more separated now.

Reply:
I was going to leave these in, but the queries wouldn't pass codegen because `Query.search` won't accept `FieldZone` or `FieldRegion` as a valid `input.type`. Seems like that was maybe missed in the API update?

Reply:
Ok we can leave them out for now. https://github.com/SeedCompany/cord-api-v3/issues/1295